@xen-orchestra/backups/_cleanVm.js

```diff
@@ -409,8 +409,64 @@ exports.cleanVm = async function cleanVm(
 
       onLog(`the VHD ${vhd} is unused`)
       if (remove) {
-        onLog(`deleting unused VHD ${vhd}`)
-        unusedVhdsDeletion.push(VhdAbstract.unlink(handler, vhd))
+        onLog(`getUsedChildChainOrDelete: deleting unused VHD ${vhd}`, {
```

Comment:
No need to duplicate data in logs.

```suggestion
        onLog(`getUsedChildChainOrDelete: deleting unused VHD`, {
```